schemas/descriptors/label/labeldescriptor.schema.json

```diff
@@ -0,0 +1,72 @@
+{
+  "meta:license": [
+    "Copyright 2021 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/descriptors/labeldescriptor",
```

Comment:
No need for this file. Simply add the "xdm:labels" field definition to the new https://ns.adobe.com/xdm/common/descriptors/descriptorLabel